packages/mui-material/src/Stack/Stack.test.js

```diff
@@ -221,6 +221,45 @@ describe('<Stack />', () => {
         display: 'flex',
       });
     });
+
+    it('should place correct margin direction even though breakpoints are not fully provided', () => {
```

Comment:
Let's also add test where more than one breakpoint is added, but the first is missing. Also, let the order be descending.